docs/site/Loopback-component-authorization.md

```diff
@@ -74,15 +74,15 @@ approach authorization.
 `Developers` need to,
 
 - mount the authorization component
-  ([see, Registering the Authorization Component](##Registering-the-Authorization-Component))
+  ([see Registering the Authorization Component](#registering-the-authorization-component))
```

Comment:
maybe` (see [Registering the Authorization Component](#..))`?

Reply:
good point. let me update that.